exonum/src/blockchain/mod.rs

```diff
@@ -117,28 +118,30 @@ impl Blockchain {
         }
     }
 
-    /// Returns service `VecMap` for all our services.
+    /// Returns the `VecMap` for all our services. This is a table which
```

Comment:
I find the use of "our" here quite strange. Also, suggest to replace "table" with "map".